pwiz_tools/Skyline/Controls/Graphs/AreaReplicateGraphPane.cs

```diff
@@ -1,1473 +1,1518 @@
-                        if (!pointPair.IsInvalid)
-                        {
-                            sumArea += pointPair.Y;
-                            maxArea = Math.Max(maxArea, pointPair.Y);
-                        }
-                    }
-
-                    // Aggregate area of this transition for each area entry
-                    Func<double,double, double> aggregateFunc = (sums, y) => sums + y;
-                    //if bar type is cluster we need to show dotp label above the highest peak, not at the total peak height
-                    if (BarSettings.Type == BarType.Cluster)
-                        aggregateFunc = Math.Max;   
-                    AddAreasToSums(pointPairList, sumAreas, aggregateFunc);
-
-                    var lineItem = curveItem as LineItem;
-                    if (lineItem != null)
-                    {
-                        lineItem.Tag = identityPath;
-                        CurveList.Add(lineItem);
-                    }
-                    else
-                    {
-                        var barItem = (BarItem) curveItem;
-                        barItem.Bar.Border.IsVisible = false;
-                        barItem.Bar.Fill.Brush = GetBrushForNode(document.Settings, docNode, color);
-                        barItem.Tag = new IdentityPath(identityPath, docNode.Id);
-                        CurveList.Add(barItem);
-                    }
-                }
-            }
-
-            ParentGroupNode = parentGroupNode;
-            SumAreas = sumAreas;
-
-            // Draw a box around the currently selected replicate
-            if (ShowSelection && maxArea >  -double.MaxValue)
-            {
-                AddSelection(normalizeOption, selectedReplicateIndex, sumArea, maxArea);
-            }
-            // Reset the scale when the parent node changes
-            bool resetAxes = (_parentNode == null || !ReferenceEquals(_parentNode.Id, parentNode.Id));
-            _parentNode = parentNode;
-
-            _dotpData = null;
+﻿/*
+ * Original author: Brendan MacLean <brendanx .at. u.washington.edu>,
+ *                  MacCoss Lab, Department of Genome Sciences, UW
+ *
+ * Copyright 2009 University of Washington - Seattle, WA
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Drawing.Drawing2D;
+using System.Globalization;

```

Comment:
It appears you change the line endings in this file. The diffs are only shown appropriately when you hide whitespace changes.

Reply:
Looks like it happened somehow during this merge from master: https://github.com/ProteoWizard/pwiz/pull/1853/commits/c0155f6ea9999f4080bf0938c37f5d6008ac8502
Current version of the file has consistent Windows style line endings (CR/LF)